readme.md

```diff
@@ -134,44 +136,62 @@ If you need to override the default values, provide the following options:
 
 ## Local authentication & authorization emulation
 
-When requesting `./auth/login/<provider>`, you will access a local authentication UI allowing you to set fake user information.
+The CLI allows you to mock and read authentication & authorization credentials.
+
+> **NOTE:** You must use port `4280` to access the authentication UI.
+
+### Mocking credentials
+
+When requesting `http://localhost:4280/.auth/login/<PROVIDER_NAME>`, you have access a local authentication UI allowing you to set fake user information.
+
+Provider names available include:
```

Comment:
User may also use their own auth providers

Reply:
okay - maybe just take this section out then?